### PR TITLE
Optimized cytoscape

### PR DESCRIPTION
**Pull request by Richard-Roberts**

### CODE REVIEW

File: client/src/components/traceView.tsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import CytoscapeComponent from 'react-cytoscapejs';
...
 import styleSheet from '../styles/Stylesheet'
-import options2 from '../constants/CytoscapeConfig'
+import options from '../constants/CytoscapeConfig'
 
 cytoscape.use(coseBilkent);
 const TraceVisualizer = (data:any) => {
-  const layout = options2()
+  const layout = options()
 
   console.log(data.data);
```
Comment (drewdunne) [inline client/src/components/traceView.tsx:12]: Can we remove the console logs  in dev unless they're needed for some reason?

File: client/src/components/traceView.tsx [hunk 2]
```diff
@@ -25,15 +25,17 @@ const TraceVisualizer = (data:any) => {
...
         objectFit: 'cover',
+        backgroundColor: '#161820'
       }}
+      maxZoom={3}
+      minZoom={0.1}
       cy={cy => {
         myCyRef = cy;
 
-        console.log("EVT", cy);
+        // console.log("EVT", cy);
```
Comment (drewdunne) [inline client/src/components/traceView.tsx:38]: Looks like this was removed but want to be sure

File: client/src/styles/Stylesheet.tsx [hunk 3]
```diff
@@ -1,114 +1,169 @@
...
+      label: 'data(label)',
+      color: 'white',
       'border-width': '1px',
-      'border-color': 'grey',
-      
+      'border-style': 'solid',
+      'border-color': '#EFEFEF',
+      'border-opacity': '1',
+      // 'text-wrap': 'ellipsis',
+      // 'text-max-width': '10px',
```
Comment (drewdunne) [inline client/src/styles/Stylesheet.tsx:43]: Are these needed too

File: client/src/styles/Stylesheet.tsx [hunk 4]
```diff
@@ -1,114 +1,169 @@
...
         "text-background-opacity": 1,
-        "color": "black",
-        "text-background-color": "whitesmoke",
+        "color": "white",
+        "text-background-color": "#161820",
+        'font-family': 'SF Pro',
     }
-  }
+  },
+  // {
```
Comment (drewdunne) [inline client/src/styles/Stylesheet.tsx:158]: probably not needed

File: server/models/ClusterModel.ts [hunk 5]
```diff
@@ -57,6 +57,7 @@ class ClusterModel {
       }
     
     res.locals.elements = elements
+    console.log(elements)
```
Comment (drewdunne) [inline server/models/ClusterModel.ts:60]: needed?

File: client/src/constants/CytoscapeConfig.tsx [hunk 6]
```diff
@@ -53,4 +108,4 @@ const options2 = () => { return {
   initialEnergyOnIncremental: 0.5}
 }
 
-export default options2;
\ No newline at end of file
+export default options2;
```
Comment (drewdunne) [inline client/src/constants/CytoscapeConfig.tsx:111]: we should probably rename this export

File: client/src/constants/CytoscapeConfig.tsx [hunk 7]
```diff
@@ -1,4 +1,59 @@
 const options2 = () => { return {
```
Comment (drewdunne) [inline client/src/constants/CytoscapeConfig.tsx:1]: probably rename this var

**Review by drewdunne — APPROVED** [review]

File: client/src/components/traceView.tsx (hunk 2 above)
Comment (Richard-Roberts) [reply client/src/components/traceView.tsx:38]: Yea, I removed the console log just to clean up the console as we're already console logging quite a bit on click to show whats received

File: client/src/styles/Stylesheet.tsx (hunk 3 above)
Comment (Richard-Roberts) [reply client/src/styles/Stylesheet.tsx:43]: No, these will actually crash cytoscape.

File: client/src/styles/Stylesheet.tsx (hunk 4 above)
Comment (Richard-Roberts) [reply client/src/styles/Stylesheet.tsx:158]: yea, we can remove. The wrapping function is currently crashing cytoscape.

**Pull request merged**